string agent_id
diagnostic_msgs/KeyValue[] local_properties
rasberry_coordination/Module[] modules
